external/fv3fit/fv3fit/reservoir/config.py

```diff
@@ -106,6 +106,7 @@ class ReservoirTrainingConfig(Hyperparameters):
     square_half_hidden_state: bool = False
     hybrid_variables: Optional[Sequence[str]] = None
     mask_variable: Optional[str] = None
+    validate_sst_only: bool = False
```

Comment:
New flag that switches between atmosphere validation and validation I've been using for sst models.  Defaults to `False` for backwards compatibility